packages/data-table-server/src/dataTableService/internal/EventsDataTableService.ts

```diff
@@ -77,7 +84,12 @@ export class EventsDataTableService extends DataTableService<
       aggregations,
     } = params;
 
-    const aggregator = new Aggregator(new DataBroker({ services: this.apiClient }));
+    const aggregator = new Aggregator(
+      new DataBroker({
+        services: this.ctx.apiClient,
+        accessPolicy: this.ctx.accessPolicy,
```

Comment:
Passing in the `accessPolicy` is the only real change here, everything else is just refactoring